### ISSUE TITLE
Main page invisible items

### ISSUE THREAD

**Issue by smcintosh881** [RITAccess/data-analyzer#158]
when you use voiceover to read the main page the Play Pause and Stop audio buttons can be seen by voice over no the eye.

**Comment by smcintosh881** [comment]
Fixed this by making the three buttons hidden using aria-hidden="true" then whenever create new table or upload cvs "submit" button is selected they become aria-hidden="false". That means they cannot be seen by voice over unless you have uploaded or created a graph.
